src/index.html

```diff
@@ -17,16 +17,24 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-wrapper">
+      <form class="search-bar search-bar--big" data-qa="big">
+        <input
+          class="search-bar__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form class="search-bar search-bar--small" data-qa="small">
+        <input
+          class="search-bar__input--small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
+    </div>
```

Comment:
You can use the `section` or `main` semantic tag instead of the `div` to wrap your form elements. This will improve the structure of your HTML and make it more accessible.